anymail/backends/mailpace.py

```diff
@@ -0,0 +1,190 @@
+from ..exceptions import AnymailRequestsAPIError
+from ..message import AnymailRecipientStatus
+from ..utils import CaseInsensitiveCasePreservingDict, get_anymail_setting
+from .base_requests import AnymailRequestsBackend, RequestsPayload
+
+
+class EmailBackend(AnymailRequestsBackend):
+    """
+    MailPace API Email Backend
+    """
+
+    esp_name = "MailPace"
+
+    def __init__(self, **kwargs):
+        """Init options from Django settings"""
+        esp_name = self.esp_name
+        self.server_token = get_anymail_setting(
+            "server_token", esp_name=esp_name, kwargs=kwargs, allow_bare=True
+        )
+        api_url = get_anymail_setting(
+            "api_url",
+            esp_name=esp_name,
+            kwargs=kwargs,
+            default="https://app.mailpace.com/api/v1/send",
```

Comment:
For consistency with other ESPs, I'd suggest just using the API base url here (`"https://app.mailpace.com/api/v1/"`), and then implementing MailPacePayload.get_api_endpoint that just returns `"send"`. ([Example](https://github.com/anymail/django-anymail/blob/v10.2/anymail/backends/resend.py#L83-L84).) Even though "send" is your only API right now, this simplifies things if you later decide to add a separate API like "send-template" or "send-batch".